cmd/cluster-agent/api/server.go

```diff
@@ -133,8 +132,10 @@ func StartServer(ctx context.Context, w workloadmeta.Component, taggerComp tagge
 	}
 
 	grpcSrv := grpc.NewServer(opts...)
+	// event size should be small enough to fit within the grpc max message size
+	maxEventSize := maxMessageSize / 2
```

Comment:
I think `2` is still a bit arbitrary to me. Isn't a way to compute the size needed dynamically ? @clamoriniere You may have more context than me on this

Reply:
We discussed with @adel121, and indeed 2 is a bit arbitrary, however the idea of the message is to container many events, so already 2 Mo is already very large. This setting is to protect the agent against edge cases  but it should never to reach.

@adel121 maybe we could add some observability to monitor the message and even sizes in order to improve later the default max size base on real data.